Tribler/Test/test_my_channel.py

```diff
@@ -156,7 +156,7 @@ def take_screenshot_quit():
 
         def check_home():
             self.guiUtility.ShowPage('home')
-            self.CallConditional(20, lambda: len(self.guiUtility.frame.home.aw_panel.list.GetItems()) == 1,
+            self.CallConditional(20, lambda: len(self.guiUtility.frame.home.aw_panel.list.GetItems()) >= 1,
```

Comment:
Only one item should be added the the `ArtworkPanel` so I think we should leave the equality check.